source/index.rst

```diff
@@ -33,6 +33,7 @@ Welcome to OpenDroneMap's documentation
     arguments
     outputs
     gcp
+    GCPaccuracy
```

Comment:
there's no standard yet. but i think we might want to change this (and the filename) to `gcp-accuracy`? the others are all lower case and the one with more than 1 word has a hyphen (`requesting-features`).

Reply:
Ok, gcp-accuracy sounds good. I can modify this files.